STYLE_API_TEMPLATE.md

```diff
@@ -52,6 +53,19 @@ If applicable, provide any caveats to its usage with a note or tip, as in the fo
 "If you use the Security plugin, make sure you have the appropriate permissions."
 (To set this point in note-style format, follow the text on the next line with {: .note})
 
+### Path and HTTP methods
+
+For relatively complex API calls that include path parameters, it's sometimes a good idea to provide an example so that users can visualize how the request is properly formed. This section is optional and includes examples that illustrate how the endpoint and path parameters fit together in the request. The following is an example of this section for the Nodes stats API:
```

Comment:
```suggestion
For relatively complex API calls that include path parameters, it's sometimes a good idea to provide an example so that users can visualize how the request is properly formed. This section is optional and includes examples that illustrate how the endpoint and path parameters fit together in the request. The following is an example of this section for the nodes stats API:
```

Reply:
It is "nodes stats API" on the website.